doctrine/mongodb-odm-bundle/3.3/manifest.json

```diff
@@ -7,6 +7,7 @@
         "src/": "%SRC_DIR%/"
     },
     "env": {
-        "MONGODB_URL": "mongodb://localhost:27017/symfony"
+        "MONGODB_URL": "mongodb://localhost:27017",
+        "MONGODB_DB": "symfony"
```

Comment:
So, it means that the bundle does not parse the DSN for database name? Wouldn't it be better to "fix" that in the bundle? Or perhaps that does not make sense for MongoDB?